solidity/package.json

```
@@ -18,9 +18,14 @@
     "test:quick": "mocha --exit --recursive --timeout 45000",
     "test:quick:watch": "mocha --watch --exit --recursive --timeout 45000",
     "demo": "truffle migrate --reset && truffle exec ./scripts/delegate-tokens.js",
-    "lint": "solium --dir ./contracts",
+    "lint": "npm run lint:js && npm run lint:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
-    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}"
+    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
+    "lint:sol": "solium -d contracts/ && prettier --list-different contracts/ **/*.sol",
+    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write contracts/ **/*.sol"
```

Why we added `contracts/` to prettier? Isn't `prettier --list-different **/*.sol` and `prettier --write **/*.sol` enough?

---

It didn't go to the nested `.sol` contracts under the `/contracts` directory. Ex. `/contracts/libraries`,  `/contracts/stubs` etc.

---

Is this space here on purpose?

```suggestion
    "lint:sol": "solium -d contracts/ && prettier --list-different contracts/**/*.sol",
    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write contracts/**/*.sol"
```

---

yes, otherwise it didn't work for me

---

Try using quotes around the path as in [an example in docs](https://prettier.io/docs/en/cli.html#--list-different).

```suggestion
    "lint:sol": "solium -d contracts/ && prettier --list-different '**/*.sol'",
    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write '**/*.sol'"
```

---

Yup, quotes worked! 68db3a8